src/qt/bitcoingui.cpp

```diff
@@ -964,6 +969,11 @@ void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate, double nVer
 
     tooltip = tr("Processed %n block(s) of transaction history.", "", count);
 
+#ifdef Q_OS_WIN
+    m_taskbar_button->setWindow(windowHandle());
```

Comment:
I don't think this is the place to do `setWindow`, every time `setNumBlocks` is called, move it to the initialization maybe?

Reply:
Dunno, this apparently works as-is at least.

(I don't run Windows, so any major changes might be better off with someone else taking over the PR.)